app/store/modules/cart/index.js

```diff
@@ -1,10 +1,110 @@
-// store/modules/cart/index.js
+import cartData from './mock/cart';
+import api from './../../utils/api';
+
+/**
+ * Finds the first occurence of item in given cart object
+ *
+ * @param {Object} cart - Cart object
+ * @param {String} sku - Sku of the cart item
+ * @param {String} groupKey - groupKey of the cart item
+ * @returns {Object} - Cart item, if found
+ */
+function findInCart(cart, sku, groupKey) {
+  return cart.items.find(i => i.sku === sku && i.groupKey === groupKey);
+}
 
 export default {
   namespaced: true,
-  state: {},
-  getters: {},
-  mutations: {},
-  actions: {},
-};
+  state: {
+    /**
+     * @type {Object}   Stores cart items
+     */
+    cart: cartData, // TODO - remove mock data once we have the api
+
+    /**
+     * @type {Object} API error
+     */
+    apiError: null,
```

Comment:
API errors should be handled by axios by default. This will be implemented with the notifications component.